packages/fs-detectors/test/fixtures/get-monorepo-default-settings/turbo/turbo.json

```diff
@@ -0,0 +1,2 @@
+// TEST COMMENT TO VERIFY JSON5 SUPPORT
```

Comment:
Comment still needed?

Reply:
yes - if we don't use a JSON5 parser then this comment will cause things to fail (and that is intentional)